Makefile

```diff
@@ -49,7 +48,11 @@ endif
 
 container: test verify-codegen nginx-ingress certificate-and-key
 	cp $(DOCKERFILEPATH)/$(DOCKERFILE) ./Dockerfile
-	docker build $(DOCKER_BUILD_OPTIONS) --build-arg IC_VERSION=$(VERSION)-$(GIT_COMMIT) -f Dockerfile -t $(PREFIX):$(TAG) .
+ifeq ($(BUILD_IN_CONTAINER),1)
```

Comment:
how about you make a different make target for `local` and call the right docker target? this `container` target is also calling `nginx-ingress` when you're not building inside the container.